oauth2_http/java/com/google/auth/http/HttpCredentialsAdapter.java

```diff
@@ -32,7 +32,6 @@ public HttpCredentialsAdapter(Credentials credentials) {
      *
      * @param request HTTP request
      */
-  @Override
```

Comment:
I get a build error in Eclipse if this is removed. I think it is because it is implementing an interface method, not overriding a virtual method.